README.md

```
@@ -174,18 +174,20 @@ Read names encode ground-truth alignment information for use by `holodeck eval`
 
 **Encoded format** (default):
 ```
-PE: @holodeck:READ_NUM:CONTIG:POS1+STRAND:POS2+STRAND:HAP:ERRS1:ERRS2
-SE: @holodeck:READ_NUM:CONTIG:POS+STRAND:HAP:ERRS
+PE: @holodeck:READ_NUM:FRAG_LEN:CONTIG:POS1+STRAND:POS2+STRAND:HAP:ERRS1:ERRS2
+SE: @holodeck:READ_NUM:FRAG_LEN:CONTIG:POS+STRAND:HAP:ERRS
 ```
 
 Example:
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)